sources/south-america/br/Rio_2013.geojson

```diff
@@ -7,13 +7,611 @@
         "country_code": "BR",
         "name": "Rio Mosaic 2013",
         "available_projections": [
+<<<<<<< HEAD
```

Comment:
hmm, strange. Ok, I think I can merge the Londrina changes manually. Don't worry.